### PR TITLE
AST-281 - fix: Secondary menu not expendable on mobile when outside of Off Canvas

### PR DESCRIPTION
**Pull request by sagarb90**
### Description
- https://wp-astra.atlassian.net/browse/AST-281
- Secondary menu not expendable on mobile when outside of Off Canvas.

### Screenshots
 https://share.getcloudapp.com/8LuoyZkY

### Types of changes
Bug fix (non-breaking change which fixes an issue).

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [x] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: assets/js/unminified/frontend.js [hunk 1]
```diff
@@ -646,17 +647,21 @@ var astraTriggerEvent = function astraTriggerEvent( el, typeArg ) {
...
 
-					if (document.querySelector('header.site-header').classList.contains('ast-builder-menu-toggle-link')) {
-						var astra_menu_toggle = __main_header_all[i].querySelectorAll('ul.main-header-menu .menu-item-has-children > .menu-link, ul.main-header-menu .ast-menu-toggle');
-					} else {
-						var astra_menu_toggle = __main_header_all[i].querySelectorAll('ul.main-header-menu .ast-menu-toggle');
-					}
-					// Add Eventlisteners for Submenu.
-					if (astra_menu_toggle.length > 0) {
+					// To handle the comma seprated selector added above we need this loop.
+					for( var k=0; k < __main_header_all.length; k++ ){
```
Comment (premanshup) [inline assets/js/unminified/frontend.js:651]: @sagarb90 please rename `k` to mainHeaderCount or something descriptive.

File: assets/js/unminified/frontend.js [hunk 2]
```diff
@@ -613,7 +613,8 @@ var astraTriggerEvent = function astraTriggerEvent( el, typeArg ) {
 	AstraToggleSetup = function () {
 
 		if ( 'off-canvas' === mobileHeaderType || 'full-width' === mobileHeaderType ) {
-			var __main_header_all = document.querySelectorAll( '#ast-mobile-popup' );
+			// comma seprated selector added, if menu is outside of off-canvas then submenu will not clickable, it work only for off-cnavas area submenu.
```
Comment (premanshup) [inline assets/js/unminified/frontend.js:616]:
```suggestion
			// comma separated selector added, if menu is outside of Off-Canvas then submenu is not clickable, it work only for Off-Canvas area with dropdown style.
```

Please add changelog as well.

**Review by premanshup — CHANGES_REQUESTED** [review]

File: assets/js/unminified/frontend.js (hunk 1 above)
Comment (sagarb90) [reply assets/js/unminified/frontend.js]: Yes done.

File: assets/js/unminified/frontend.js (hunk 2 above)
Comment (sagarb90) [reply assets/js/unminified/frontend.js]: Comment updated also added changelog for the same just confirm the version that I have added.

File: assets/js/unminified/frontend.js [hunk 3]
```diff
@@ -646,17 +647,21 @@ var astraTriggerEvent = function astraTriggerEvent( el, typeArg ) {
...
-					} else {
-						var astra_menu_toggle = __main_header_all[i].querySelectorAll('ul.main-header-menu .ast-menu-toggle');
-					}
-					// Add Eventlisteners for Submenu.
-					if (astra_menu_toggle.length > 0) {
+					// To handle the comma seprated selector added above we need this loop.
+					for( var mainHeaderCount =0; mainHeaderCount  < __main_header_all.length; mainHeaderCount++ ){
+
+						if (document.querySelector('header.site-header').classList.contains('ast-builder-menu-toggle-link')) {
+							var astra_menu_toggle = __main_header_all[k].querySelectorAll('ul.main-header-menu .menu-item-has-children > .menu-link, ul.main-header-menu .ast-menu-toggle');
```
Comment (premanshup) [inline assets/js/unminified/frontend.js:654]: @sagarb90 Have we not updated the `k` from everywhere? Please make sure to test this once after updating.

**Review by premanshup — CHANGES_REQUESTED** [review]

File: assets/js/unminified/frontend.js (hunk 3 above)
Comment (sagarb90) [reply assets/js/unminified/frontend.js]: @premanshup Opps my bad I missed it, now I have modified and tested the changes as well. thank you.

Comment (premanshup) [reply assets/js/unminified/frontend.js]: @sagarb90 No worries 🙂

**Review by premanshup — DISMISSED** [review]

**Review by abhishek-bsf — APPROVED** [review]

**Pull request merged**